code/src/styles/MovieList.css

```diff
@@ -0,0 +1,125 @@
+
+* {
+  box-sizing: border-box;
+}
+
+.list-movie-wrapper {
+  display: grid;
+  grid-template-columns: repeat(5,1fr);
+  grid-auto-rows: auto;
+  grid-gap: 0;
+  grid-row-gap: 0;
+  row-gap: 0;
+  justify-items: center;
+  justify-content: center;
+  width: 100vw;
+  background: #000;
+}		
+	
+.list-movie-item {
+  position: relative;
+  max-height: 100%;
+}
+
+.list-movie-poster {
+  width: 100%;
+  height: 100%;
+}
+
+.list-details-wrapper {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Right now this does not do anything, since you are using margin top 50% on your h2 tag. With long titles the text is squeezed and hard to read. You could remove the margin top on that tag and maybe use "space-evenly" instead to align hover text in center.